### ISSUE TITLE
Allow inspecting GraphQL schema documentation from a query

### ISSUE THREAD

**Issue by nickserv** [a85/POSTMan-Chrome-Extension#11041]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

I find it significantly more difficult to write GraphQL requests in Postman than other tools because there seems to be no way to look up GraphQL schema documentation from my query code. Creating an API in Postman is not very helpful because I would still have to manually find the documentation for every query and mutation I'm using.

### Describe the solution you'd like

GraphiQL (the official query editor for GraphQL) solves this by letting you click on any property or type to reveal its documentation in the sidebar. Postman should already have the schema as it supports introspection, but ideally it should be accessible in the editor. It should be possible to use GraphiQL for this internally, as it's permissively licensed. GraphiQL also doesn't require you to manually import an API to view its documentation.

I'd also like GraphQL-friendly documentation support, though that's less important to me because the APIs I'm using already have good documentation online.

### Describe alternatives you've considered

Using another GraphQL editor like GraphiQL or Insomnia

### Additional context

Inspired by https://github.com/postmanlabs/postman-app-support/issues/6719#issuecomment-1144933008

**Comment by JulianGuSM** [comment]
It would be great nice if Postman support fetching GraphQL schema documentation from a query and show it at sidebar or somewhere else. By now, I use GraphQL Playground to call GraphQL request, however, switching between different apps is a bit of a hassle.

**Comment by jagrutmehta** [comment]
Yes. Explanatory documentation is very helpful to understand what this field means and what it represents. Currently I only have field name while writing query.

**Comment by ItsChrisHickman** [comment]
This would be highly useful as I alternate between GraphiQL, Altair and Postman for this very reason and would prefer to do everything within Postman.

**Comment by bezreyhan** [comment]
I just started with Postman and GraphQL and was searching for this feature for a while. Being able to see the full schema is very helpful. I currently need to have both Postman and GraphiQL open and it would be helpful to just be able to use Postman.

**Comment by jktravis** [comment]
We’ve recently started to use GraphQL, and my QA team is always asking for documentation for the new operations. I keep telling them it’s self-documenting and how to use the autocomplete, but that doesn’t seem to be adequate. I end up having to export a collection for them.

**Comment by coderite** [comment]
+1 if this feature is implemented Postman becomes much more convenient than apps like GraphiQL.

**Comment by nzbart** [comment]
People in my team are already familiar with - and like - Postman, but I have to tell them they are better off learning another tool such as [Banana Cake Pop](https://chillicream.com/docs/bananacakepop) for GraphQL APIs because the schema discovery and presentation is a lot better.

**Comment by NajeebTyson** [comment]
When I first started using Postman for GraphQL the first thing I wanted to see was the schema and got irritated while searching for it in postman. How can the team miss such a big feature, in fact, this is supported by all other tools but they are not as good as Postman.
Whats the update on this?

**Comment by NourNafea** [comment]
It's 2023 and we still cannot see the schema

**Comment by SterlingChin** [comment]
The new GraphQL client is released and in [open beta](https://blog.postman.com/announcing-postmans-new-graphql-client/). The new client automatically imports your schema through introspection, or you can load it from an existing API or import it from your local machine. The schema explorer on the left side provides the ability to see the schema and the description of the fields. You can also turn the description on/off as necessary.

## Hide/Show descriptions with Introspecton
![graphql-hide-show-descriptions](https://user-images.githubusercontent.com/8900638/221945220-3530bf23-fc83-4a32-9df2-1a0c14638c16.gif)

## Descriptions loaded from local import
![graphql-import-documentation](https://user-images.githubusercontent.com/8900638/221945422-16065f42-a019-46ca-8c76-26894e6c4c86.gif)

**Comment by jagrutmehta** [comment]
@SterlingChin I got 10.12.x client. Under `Documentation`, I can see raw schema but without any field/query comments/documentation. Am I missing something? Without field/query level comments/documentation, it is very tough to build query.

**Comment by SterlingChin** [comment]
We are currently working on a documentation feature for the beta client. I'd love to hear your feature needs in greater detail so we can work toward a solution that users will enjoy.

**Comment by jagrutmehta** [comment]
> We are currently working on a documentation feature for the beta client. I'd love to hear your feature needs in greater detail so we can work toward a solution that users will enjoy.

Two asks

1. Navigable documentation - just like `graphiql` or other graphql clients like `altair`. It starts with high level clickable query/mutation and then user can navigate to individual types and requests. 
2. Actual documented comments for each query and fields.

With above combined with other postman goodies will make it very powerful graphql client.

**Comment by ItsChrisHickman** [comment]
The GraphQL query builder is fantastic.  I would prefer it to be on the right, not left 😛.  It doesn't feel right on the left...

Additionally, at my company, we are able to use tokens in headers to authenticate to our graph endpoints.  This is useful in postman and works with http requests, but not graphql ones.  Could the authentication method to inherit be available here too?

You guys are doing an awesome job on this integration!

**Comment by nkhil** [comment]
> We are currently working on a documentation feature for the beta client. I'd love to hear your feature needs in greater detail so we can work toward a solution that users will enjoy.

Feature parity with [Insomnia](https://insomnia.rest/) client would be great. Currently, it allows a 'Show Documentation' button that displays the schema to explore. This helps while creating queries and mutations.

**Comment by Max101** [comment]
Is this true? In 2023 postman does not allow to see the schema of a graphQL api? 🤦

**Comment by giridharvc7** [comment]
@Max101  
Have you tried our new GraphQL Client? You can try this out like so

https://github.com/postmanlabs/postman-app-support/assets/1040583/6a5e1c43-05b3-4166-8bbd-8127660a4ad6

Ensure you are on the latest version of the app!

**Comment by kevmul** [comment]
I have just discovered your GraphQL Client and it looks like it would be really great. However, I get an error when trying to access my AWS bucket. I have all my auth setup and everything works to query the server on HTTP using GraphQL. Just not the GraphQL Client
```
{
    "errors": [
        {
            "errorType": "BadRequestException",
            "message": "The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details.\n\nThe Canonical String for this request should have been\n'[REDACTED]\n"
        }
    ]
}
```
![image](https://github.com/postmanlabs/postman-app-support/assets/5790104/5f434ff2-ede8-464c-b572-2dde5c3b68dc)
